.github/workflows/job_build_linux.yml

```diff
@@ -229,6 +233,18 @@ jobs:
           cmake --build ${BUILD_DIR} --parallel
           cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR_JS} -P ${BUILD_DIR}/cmake_install.cmake
 
+      - name: Build RPM packages
+        if: ${{ inputs.build-rpm-packages }}
+        run: |
+          [[ ${{ inputs.os }} == "fedora_29" ]] && python_exec=/venv/bin/python3 || python_exec=/usr/bin/python3
```

Comment:
looks like WA

does not `RUN alternatives --install /usr/bin/python3 python3 /usr/local/bin/python3.8 1` work as expected?

Reply:
Unfortunately, it does not. 

Before I tried this with `alternatives`, for some reason, it had searched for Python **3.8** in `/usr/bin` while there is python3.7 binary in this directory like in https://github.com/openvinotoolkit/openvino/actions/runs/10737772205/job/29780259018.
Now, with this WA (`alternatives` + pointing to venv), the error is:
![image](https://github.com/user-attachments/assets/ea95a596-e699-4837-bcba-11eaa33aac61) so I guess it does not like symlinks/Fedora does not like being interfered with its default Python.

The question is, why does it try to use py3.8, while the default Python is 3.7. Is it because the cmake option with the Python executable is provided?

Reply:
This WA also breaks yum/dnf and, most likely, most of the other Python-dependent packages.